dali/operators/generic/erase/erase.cu

```diff
@@ -99,7 +99,7 @@ class EraseImplGpu : public OpImplBase<GPUBackend> {
   }
 
  private:
-  const OpSpec &spec_;
+  const OpSpec spec_;
```

Comment:
Does it make sense to add a test for that?

Reply:
Do you have a good idea how to cover all ops without having to write a lot of code? The issue happens during run.

Reply:
Not really. If you don't have any let us skip it.

Reply:
Please revert. This class is safe - it's a reference to a member of the enclosing object - the lifetime of the real object is longer than that of this class.